code/game/objects/items/ego_weapons/aleph.dm

```diff
@@ -404,3 +404,104 @@
 
 /obj/item/ego_weapon/censored/get_clamped_volume()
 	return 50
+
+/obj/item/ego_weapon/gunblade
```

Comment:
Wouldn't it be better to refactor this as a gun? Just asking.

Reply:
Honestly, not really sure